src/wizards/audience/views/setup/index.js

```diff
@@ -99,10 +99,11 @@ function AudienceWizard( { confirmAction, pluginRequirements, wizardApiFetch },
 			label: config.enabled ? __( 'Configuration', 'newspack-plugin' ) : __( 'Setup', 'newspack-plugin' ),
 			path: '/',
 		},
-		config.enabled && {
-			label: __( 'Content Gating', 'newspack-plugin' ),
-			path: '/content-gating',
-		},
+		config.enabled &&
+			config.has_memberships && {
+				label: __( 'Content Gating', 'newspack-plugin' ),
+				path: '/content-gating',
+			},
```

Comment:
`has_memberships` is a localized value (`window.newspackAudience.has_memberships`), not part of this config payload.

The Content Gating tab is never rendering.